Backend/scraper/updateServer.js

```diff
@@ -0,0 +1,51 @@
+let Platform = require('../models/platform.model');
+let Show = require('../models/show.model');
+
+/*  To use this function, make sure the parameters are as follows:
+
+        anime_list - [{title:<string>}] - Later this can include genre and stuff like that, so for scalability
+                                          this is an array of objects instead of an array of strings.
+        target_platform = {websiteName:<string>, link:<string>} 
+
+
+    How to import: import { updateServer } from "updateServer";
+
+    Example of use: 
+        let ridaas_anime_list = [{title:"My Hero Academia"}, {title:"Naruto"}, {title:"JoJo's Bizzare Adventure"}];
+        let ridaas_target_platform = {websiteName:"Crunchy Roll", "www.crunchyroll.com"};
+        
+        updateServer(ridaas_anime_list, ridaas_target_platform);
+*/      
+
+let updateServer = (anime_list, target_platform) => {
+    
+    Platform.findOne({websiteName:target_platform.name})
+    .then(platform => {
```

Comment:
@ChrisHaliga Looks good. Just one minor issue it would be target_platform.websiteName (based on your description of parameters in the comments).